### PR TITLE
Mention Auto-registration of New Content Element Wizard

### PR DESCRIPTION
**Pull request by schliesser**
Update documentation to reflect changes made to TYPO3 in v13.0: Feature: #102834 - Auto-registration of New Content Element Wizard via TCA

### CODE REVIEW

**Comment by linawolf** [comment]
Thanks a lot for your change. This references

References https://github.com/TYPO3-Documentation/Changelog-To-Doc/issues/801

Once we have updated the TCA reference I would like to link there instead of the changelog.

File: Documentation/ExtensionArchitecture/Extbase/Reference/FrontendPlugins.rst [hunk 1]
```diff
@@ -91,27 +91,9 @@ Use the following steps to add the plugin as content element:
 
 #.  (Optional) Add to the :guilabel:`New Content Element` wizard
```
Comment (linawolf) [inline Documentation/ExtensionArchitecture/Extbase/Reference/FrontendPlugins.rst:92]:
```suggestion
#.  Add to the :guilabel:`New Content Element` wizard (automatic)
```

File: Documentation/ExtensionArchitecture/Extbase/Reference/FrontendPlugins.rst [hunk 2]
```diff
@@ -91,27 +91,9 @@ Use the following steps to add the plugin as content element:
...
 
-    Add the following :ref:`page TSconfig <t3tsconfig:setting-page-tsconfig>`
-    to add the new plugin to the wizard:
-
-    ..  literalinclude::  _FrontendPlugin/_page.tsconfig
```
Comment (linawolf) [inline Documentation/ExtensionArchitecture/Extbase/Reference/FrontendPlugins.rst:97]: This file  is not used anymore and needs to be deleted. Can you do that or should I do it in a follow-up?

File: Documentation/ExtensionArchitecture/Extbase/Reference/FrontendPlugins.rst (hunk 1 above)
Comment (schliesser) [reply Documentation/ExtensionArchitecture/Extbase/Reference/FrontendPlugins.rst]: Done

**Comment by schliesser** [comment]
Thanks you @linawolf for the review.

Changed the text and deleted the file.

**Review by linawolf — APPROVED** [review]

**Comment by linawolf** [comment]
Thank you for working on this!

**Pull request merged**